PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -19,6 +19,19 @@
   <change-notes>
     <![CDATA[
     <html>
+      <h3>[3.12.0]</h3>
+      <ul>
+        <li>Updated to list both Windows and Linux web apps in web apps explorer.</li>
+        <li>Added support to choose a Linux web app to deploy the application.</li>
+        <li>Added support to create a new Linux web app to deploy the application.</li>
+        <li>Added support to show all the deployment slots for a web app.</li>
+        <li>Refactored web apps icons.</li>
+        <li>Refactored the use existing web apps UI when users click 'Run on Web App'.</li>
+        <li>Updated to list all app service plans of the selected subscription for users to choose to create a new web app.</li>
+        <li>Updated to use zipDeploy to upload none war files.</li>
```

Comment:
And please find the official name instead of `zipDeploy`, it looks like the name for an API.  I see [this ](https://blogs.msdn.microsoft.com/appserviceteam/2017/10/16/zip-push-deployment-for-web-apps-functions-and-webjobs/) but I'm not sure whether it is. @jdneo Do you know its official name?

Reply:
This will be removed.